sdk/JavaScript/TestFramework/js/testFramework.js

```diff
@@ -36,15 +36,15 @@ function createZumoNamespace() {
         this.canRunUnattended = true;
         this.logs = [];
         this.requiredFeatures = [];
-        if (requiredRuntimeFeatures) {
-            if (typeof requiredRuntimeFeatures === 'string') {
-                requiredRuntimeFeatures = [requiredRuntimeFeatures];
+        if (requiredRuntimeFeatures) {
```

Comment:
whats with the whitespace changes?  Are you converting tabs to spaces or changing line endings?